src/Compute/Compute.Test/ScenarioTests/DedicatedHostTests.ps1

```diff
@@ -40,15 +40,13 @@ function Test-DedicatedHost
         Assert-AreEqual $loc $hostGroup.Location;
         Assert-True { $hostGroup.Tags.Keys.Contains("key1") };
         Assert-AreEqual "val1" $hostGroup.Tags["key1"];
-        Assert-True { $hostGroup.Zones.Contains("1") };
+        #Assert-True { $hostGroup.Zones.Contains("2") };
```

Comment:
Should this be commented out?

Reply:
@haagha should know about it because he made the change haha. I guess it should be if the test pass ;). What do you say @haagha ?

Reply:
I might have temporarily commented out, can be put back in zone is not a troubling property.

Reply:
it may be because -Zone expects list of string <string[]>

Reply:
added zone assert